ocaml/typing/jkind.ml

```diff
@@ -873,24 +1034,30 @@ let get_required_layouts_level (context : History.annotation_context)
       ( Value | Immediate | Immediate64 | Any | Float64 | Float32 | Word
       | Bits32 | Bits64 ) ) ->
     Stable
-  | _, Void -> Alpha
+  | _, (Any_non_null | Value_or_null | Void) -> Alpha
 
 (******************************)
 (* construction *)
 
 let of_new_sort_var ~why =
```

Comment:
This exported function probably needs to expose the nullability choice to callers.

Reply:
This is implemented in the next PR.

Reply:
See `(* CR layouts v3.0: split those functions in two versions, one [or_null] and one [non_null]. *)`.